islamapp/model/data_models/user_followers.py

```diff
@@ -0,0 +1,26 @@
+from mongoengine import fields, Document
+
+class UserFollowers(Document):
+    
+    scraped_user_id= fields.IntField(primary_key=True,required=True)
```

Comment:
Who is the "user" mentioned here?
If it represent user of ISLAM, it can't be PK, cause a user can trigger multiple tasks.
If it's user of IG in the other hand, that user might not register in ISLAM, so we won't have user_id for it.
Consider change it into user_id of IG?

Reply:
User represent ig user . Originally, I thought that the user_id is auto incremental by data base and user is their ig Id, but it seems to be confused. Combining them together would be better.